docs/demos/video-links.md

```diff
@@ -0,0 +1,9 @@
+# Introduction and Demo Videos
+
+There are several on-line videos available to help you quickly understand the concept of Akri and get some quick demos.
+
+1. [Bridge Your IoT Leaf Devices to Local Clusters with Ease Using Akri and Dynamic Resource Allocation](https://www.youtube.com/watch?v=ZZfDne7gMMI) - Latest Akri introduction at KubeCon EU 2024.
+1. [Akri is dope!](https://www.youtube.com/watch?v=KLA3N8xbYPE) - A talk in the Azure Arc Jumpstart channel. Includes a demo of discovering an ONVIF camera with Akri and feeding the stream to a edge AI model.
+1. [Connecting to devices at the edge with minimal footprint using AKS Edge Essentials and Akri on WASMs](https://www.youtube.com/watch?v=L6IhonIUOKU) - This talk goes over how users can use AKS Edge Essentials with WASM workloads to deploy a cluster on “light” edge hardware (PC-class devices) and run Akri, minimizing the footprint when connecting to these devices and utilizing them in your applications.

```

Comment:
I wonder if we should avoid having 2 of the 3 videos be Azure focused. If we do that, we should also have a SUSE one.

Reply:
can we add the newest kubecon talk with SUSE to the list of videos: https://youtu.be/ZZfDne7gMMI?si=DPwrY2VdwSP4-y4E

Reply:
> can we add the newest kubecon talk with SUSE to the list of videos: https://youtu.be/ZZfDne7gMMI?si=DPwrY2VdwSP4-y4E

The current #1 is the kubecon talk.

Reply:
> I wonder if we should avoid having 2 of the 3 videos be Azure focused. If we do that, we should also have a SUSE one.

I'll check with Nicolas for SUSE's application. If there is none, we can remove one of the Azure related link.

Reply:
There are older ones that provide a good overview of the project that Edrick, Jiri, or I have given that come up with a "Akri cncf" YouTube search. But i am not sure what our video selection criteria is here. Is it to put all the videos or one from each area of Akri?